constants/constants.ts

```diff
@@ -151,6 +151,14 @@ export const SPELL_OWNER_ADDRESS = {
   [CHAINID.ETH_MAINNET]: "0x2ee555c9006a9dc4674f01e0d4dfc58e013708f0",
 };
 
+export const SAMB_ADDRESS = {
+  [CHAINID.ETH_MAINNET]: "0x278a8c33Ef2A7D7C682f458C85E9447ce3645188",
+};
+
+export const SAMB_OWNER_ADDRESS = {
+  [CHAINID.ETH_MAINNET]: "0x6d5b28659b6e76c5843c963f44158c3be6ede3a1",
```

Comment:
SAMB_OWNER_ADDRESS is a misnomer - its used as an address that holds a significant amount of the token so that for testing we can simulate transfers from that address to our own test addresses to play around with the token as we see fit